docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -309,8 +309,8 @@ This archive collects the following information:
 
 Note that the diagnostics bundle is intended for debugging purposes only, its structure may change between releases.
 
-IMPORTANT: The configuration files in the archive may have credentials in *plain text*.
-These will need to be manually redacted before the archive is shared.
+IMPORTANT: {agent} attempts to automatically redact credentials and API keys when creating diagnostics. 
+Please review the contents of the archive before sharing to ensure there are no credentials in plain text.
```

Comment:
```suggestion
Please review the contents of the archive before sharing to ensure that there are no credentials in plain text.
```

Reply:
Please consider adding "that" for easier parsing, especially by international audiences

Reply:
@nimarezainia, will you please give this a 👍 so we can get it merged?  Looks good from a docs POV.